src/index.css

```diff
@@ -59,7 +72,7 @@ p {
     float: right;
     height: 2vw;
     box-sizing: border-box;
-    background-color: white;
+    background-color: rgb(190, 215, 190);

```

Comment:
Again, colors should be done using exclusively hex rgb values (prefixed with #) or the rgb(), either is fine but you should not mix styles, pick one and be consistent.